packages/subgraph/tests/payload.test.ts

```diff
@@ -138,6 +135,8 @@ test("(RegisterNetworks, SetBlockNumbersForNextEpoch, SetBlockNumbersForNextEpoc
 
   processPayload(submitter, payloadBytes, txHash);
 
+  // To Do add asserts
+
```

Comment:
These 2 lines probably slipped on a rebase (might also be my fault, can't remember if I removed them 😅 )